### ISSUE TITLE
Error on Kubuntu 14.04

### ISSUE THREAD

**Issue by xstable** [martin-helmich/typo3-ci-example#1]
This line: 
`curl --insecure https://www.npmjs.org/install.sh | bash`
gives me an error while installation process:

``````
TASK: [Install NPM] *********************************************************** 
failed: [default] => {"changed": true, "cmd": "curl --insecure https://www.npmjs.org/install.sh | bash", "delta": "0:00:05.204007", "end": "2015-02-13 16:48:53.789477", "rc": 127, "start": "2015-02-13 16:48:48.585470", "warnings": ["Consider using get_url module rather than running curl"]}
stderr: bash: warning: setlocale: LC_ALL: cannot change locale (de_DE.utf8)
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100    51  100    51    0     0      9      0  0:00:05  0:00:05 --:--:--   408
bash: line 1: Moved: command not found

FATAL: all hosts have already failed -- aborting

PLAY RECAP ******************************************************************** 
           to retry, use: --limit @/home/suther/provision.yaml.retry

default                    : ok=4    changed=3    unreachable=0    failed=1   

Ansible failed to complete successfully. Any error output should be
visible above. Please fix these errors and try again.```

Any Idea why?

(I've seen ubuntu links /bin/sh to /bin/dash, maybe cause of this??)
And I use ansible inside of vagrant with debian7 (http://privatepaste.com/c80f8b7c2b). Same result. Debian also link do /bin/sh to /bin/dash
``````

**Comment by beathorst** [comment]
Long time ago but for me this works:

```
curl -L https://www.npmjs.com/install.sh | sh
```

**Comment by martin-helmich** [comment]
Oh yes, I'd lost track of this one. Thanks for the suggestion, fixed in c55474b.

**Comment by beathorst** [comment]
Perfect. Thanx
